mite_selenium/__init__.py

```diff
@@ -1,12 +1,14 @@
 import logging
+import socket
 import time
 from contextlib import asynccontextmanager
 from functools import wraps
 
 from selenium.common.exceptions import TimeoutException
-from selenium.webdriver import Remote
+from selenium.webdriver import Remote as SeleniumRemote
 from selenium.webdriver.support import expected_conditions as EC
 from selenium.webdriver.support.ui import WebDriverWait
+from seleniumwire.webdriver import Remote as SeleniumWireRemote
```

Comment:
It looked like in other places that you were checking if selenium wire existed, is this not going to break if it doesn't?

Reply:
selenium-wire is now a requirement in the dependencies in setup.cfg so It should always get installed as a dependency and always be there for import. My checks were to see if the remote object was to be a selenium or selenium-wire instance.